### ISSUE TITLE
Inserts extra newline everytime formatter runs

### ISSUE THREAD

**Issue by dbrattli** [fsprojects/fantomas#1346]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAXAngB0WAhgHbwDKiAjgDwDk6AfGALwA6hEEAPmAILFnlhYAMzC0wAZwqt2YaewCWhdIgBOQ-NDwBJAKKEArgFtV%2BAEYAbRDXpgA7vPQALOTIjHDp1WCfzxAOgBxRHQ9IxN0WBUACgBKNABhWHNLaHR5WEJ-XQNjFXwIlSYXVxlDfOhHb0dfOwdnNhLXLl5SCjAAD3EwAFoGTsDg0Nz8yNiwFAYAfUgAGnAoSXQ0wgBzcVQwAG15iFASyAASLFwNyEV0WZ32Q4A3fHN9ZDRtxrAABjnXyC1iRCUSeQAL2Qn0aABYrhAALpXAC%2BoN2kKgRxwTyg50u%2BwOdweaJejQAjAjXJAALL4doAGUUiEpfxWTkxhIATG8kTCZPCrnsSSiTmhIKZYEkmTJbvdHht8SVmcSxWRDPI4OYMtwQsQAPJCamEEFIiDqcySdlw4k8sV8tGC4XmUXXbESvH6sAAZjl1xI2A0iAAQoghJFEAAFfB5YzKFR2mToFSPE2cs1Iw7HK1CkXu8W4qXOsHuiCQT3ev0BlS02C2VTQfCSH43WBVtIZKPsGNxkoc9hcmTm%2B0p05p20Zh1Z57OgCseaghc0xcDAFVsLgVFWa4Q6w30oRmwb7sb26buUnLf2bc3M5LR68AGyTgtemf%2BwPxczV8SJTKt1KRbdgQ171wdhAXbsD2%2BbHgKA5nsOF5bM6ADst7Tr6j6lqSiAeKoP5-og8adomWJ9hBp5DjiMHSq4AAciH3shJaIIkKpbpO2G4UB%2BG8oRUCQSRjrZq8ACc1HetwQgRokhhlD%2BrY4fuCaHgRqInumR6kU6rwEh8zp3kWKGIAqSpJE2zG7jJAEHt2R6cdaylYqpfGEkSWlISJEb6cqRnOtJrFgMBiIKfyXHESpvGXiyt4-EgSgaoQAAqKiYAA6nUWEmd5vlgKByJWdxwUjrB6luk5NHcCosD6MQAAiiDmPIioRlJsamTIgE%2BexFpWRiPF5eRMgErmWnku0WhCDFjh-DoRp6Y4kToEl8CMpOYJsrJeHyRxikCp1uVkc6BITgNFI-EI8jtBqS4jCoOjtNgpbiOIm4-mOy1mXJFn%2BVaW22SF%2BWEjeB3tAASogcAqPAc0LTmz3NeZIGWRt6JKFBdmhSUBIIf9QMg-AAByRieCoWpaMohjrJOBJpW1vbw5Ahj6OYaQ1bqABikRlEsqgxYpXU7epVFaZjkTwKSdMMzSLMqGz9W3hUoYaBGkz2PNjiQBTa3tdTn28sjP2o4J-0lXkmAaio1LiLN8hKz%2BS2q2960BWciPc2pjSsreg0G-gRsm746C4xhBNCET6Gk7tNuw%2B9py0-T8iM4g4uSxzXPbc7MqOV8Hte6b6DC9Hsfx-kUtaTLeSpKoCsW4yYd%2BXbH2O8n9kyrK-0AGqOj6ijwIoKzg8ri1Q%2BwLXpZlyYa3XX3dc6zKFV8g1M%2BVqSbu3xBdz3Vv99CMPV%2Br9uaxa2s9ewzL9TPFIVbA6BBCE123fdGSr5OT1VxlcP2zl488y7%2B0zyLMc0j6KrQAANY%2BhLoA4I4hookHwMYBiRgmLOhYitNiasqavyCu-FOrhmR-S%2BNjRAthY5%2BnQBWP4nNcBVWOoQBwm4WhoX9iHV4XkkGtRQWBbK6CtbfQPhAZk6MvgAGlECIGwMNUafwfhQOMDqPUrxEEvVWrbbetcLhOwblgvmJ92gTUVOIRw99IZP2HuBQKNlOET1eMyPWXwASrEsCVFYYQlAJUQKYUkCAZGNDkdDV64ca5KUHPXFGrgXSaXTjVFYhA56EAXhkAEET8j6FLDFWAEU-joHyA9Yyk1DEv1TBwveXDnQujTo0SA3B6aqEIBkm4ZZVh0PxhQxQ1CMgMM8alZhQ9cn%2BKRoU14Lom7f2jtIyk0DTDwHwM%2BWA91Vh4IITSFK2SOmUzYdTN%2BZiP4lBdNPUpFVfBmEsNo3wjgSCYCUBSBZ-5vEKN8Uo04fx4BamkSQLAlgenmMaC6Y%2BpS9APO1PM6WKhzBCBVks1hWVVn5PtPvIpX9SnPJUEqbO7iLlNQHnCeYUJQRnHEEze6GwWKwiAA)

#### Code

```fsharp
type andSeq<'t> =
    | AndSeq of 't seq
    
    interface IEnumerable<'t> with
        member this.GetEnumerator(): Collections.IEnumerator =
            match this with
            | AndSeq xs -> xs.GetEnumerator() :> _
```
            

#### Result

```fsharp
type andSeq<'t> =
    | AndSeq of 't seq


    interface IEnumerable<'t> with
        member this.GetEnumerator(): Collections.IEnumerator =
            match this with
            | AndSeq xs -> xs.GetEnumerator() :> _

```
            
#### Problem description

Fantomas will add an extra newline every time formatter runs. In the online tools you will only see one extra newline, but if you copy the output back to the input it will keep adding newlines.

Fantomas should consider running formatter twice in tests to see that the output is stable and will not change every time the formatter runs.

Fantomas v4.4.0-alpha-007

#### Extra information

- [ ] The formatted result breaks by code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas Master at 12/26/2020 15:00:23 - 76fa31efcc568685abafe6efe770557f66147917

```fsharp
    { config with
                NewlineBetweenTypeDefinitionAndMembers = true }
```

<sub>Did you know that you can ignore files when formatting from fantomas-tool or the FAKE targets by using a [.fantomasignore file](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#ignore-files-fantomasignore)?</sub>

**Comment by knocte** [comment]
My team is also affected by this, @nojaf it would be great to get a new alpha :)

**Comment by nojaf** [comment]
I've published [4.4.0-alpha-008](https://www.nuget.org/packages/fantomas-tool/4.4.0-alpha-008).
